WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -0,0 +1,146 @@
+import SwiftUI
+
+struct ThemesPreviewView: View {
```

Comment:
Nit: Friendly reminder to add comment for any new type.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/11452/commits/8a44b7bb6c8d282a8cefd050af0a5774dd1ceecb